src/values.rst

```diff
@@ -226,10 +226,7 @@ The following :t:`[construct]s` are :t:`[variable]s`:
   A :t:`temporary`.
 
 * :dp:`fls_81dlbula47nu`
-  A named :t:`binding`.
-
-* :dp:`fls_adqfhc5k051x`
-  A named :t:`function parameter`.
```

Comment:
A function parameter introduces bindings which are the variables, I wouldn't consider the parameters themselves as variables hence